ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.1.yaml

```diff
@@ -79,10 +79,9 @@ releases:
       channel: fast
       version: "4.10"
   intermediate411:
-    candidate:
+    release:
       architecture: amd64
-      product: ocp
-      stream: nightly
+      channel: fast
```

Comment:
Hmm, this is a stable -> nightly upgrade ci, so I think we still should keep the origin setting, I think @shellyyang1989 's [comments](https://github.com/openshift/release/pull/50345#discussion_r1556819675) only talked about stable -> stable upgrade ci.

Reply:
Oh, ignore it, the 4.11 is already EOL.